client/src/components/Cart-icon/cart-icon-logo.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
importing `svg` files requires special webpack config, therefore, I converted svg file into a component and leverage it accordingly.